src/components/pages/AdminDashboard/DayComponents/DayComponents/01_Sat-Mon.js

```diff
@@ -0,0 +1,155 @@
+
+  const findDayOfWeekReference = 1;
+
+  // const handleSim = () => {
+  //   push(`${gameStageUrl}`);
+  // };
+
+  const findNextDayOfWeek = selectedDay => {
+    let date = new Date();
+    let resultDate = new Date(date.getTime());
+    resultDate.setDate(
+      //.setDate is a JS date function NOT the setDate() action used by Redux
+      date.getDate() + ((7 + selectedDay - date.getDay()) % 7)
+    );
+    return resultDate;
+  };
+
+  useEffect(() => {
+    setDate(findNextDayOfWeek(findDayOfWeekReference));
+  }, [findDayOfWeekReference]);
+
+  const handleGetChildTasks = e => {
+    getChildTasks(authState, tasks.child_id, tasks.story_id);
+  };
+
+  const handleAllTasks = e => {
+    setAllTasks(
+      authState,
+      tasks.submissionID,
+      tasks.hasRead,
+      tasks.hasWritten,
+      tasks.hasDrawn
+    );
+  };
+
+  const onSimulate = e => {
+    if (handleAllTasks() !== []) {
+      handleGetChildTasks();
+    }
+  };
+
+  const completeOneTask = e => {
+    setHasRead(true);
+    setHasDrawn(false);
+    setHasWritten(false);
+  };
+
+  const completedTwoTasks = e => {
+    setHasRead(true);
+    setHasDrawn(true);
+    setHasWritten(false);
+  };
+
+  const completedAllTasks = e => {
+    setHasRead(true);
+    setHasDrawn(true);
+    setHasWritten(true);
+  };
+
+  return (
+    <Layout>
+      <Header className="ant-page-header">
+        <h1>Story Squad</h1>
+        <Button onClick={adminDash}>Back to Admin Dashboard</Button>
+      </Header>
+      <div className="dev-tools-day">
+        <h2>Saturday / Sunday / Monday</h2>
+        <h3>Game Play Days: 1, 2, and 3</h3>
+        <h4>The Reading, Writing, and Drawing Stages</h4>
+        <Content>
+          <p>
+            <b>Saturday:</b> At 11AM every Saturday, the game is reset. A kid
+            user reads the excerpt from the story for the week. Only when the
+            kid user is finished reading the story for the week they are able to
+            proceed to the next stage of the game.
+          </p>
+          <p>
+            <b>Sunday:</b> Kid users are instructed to draw a picture of one
+            sentence from the excerpt of the story of the week. When the kid
+            user uploads the image, they proceed to the next stage in the game.
+          </p>
+          <p>
+            <b>Monday:</b> Kid users are instructed to write a side quest
+            (story) based off of the prompt given in the app. When the kid user
+            uploads the image, they proceed to the next stage in the game.
+            Monday at 11:59PM is the deadline for submissions. The image of the
+            uploaded side quest is translated by a 3rd Party API implemented by
+            DS.
+          </p>
+          <p>Select the game state you would like to see in play:</p>
+          <div className="state-buttons">
+            <Radio.Group>
+              <Radio className="radio-buttons" onChange={completeOneTask}>
```

Comment:
It looks like the reason that these all appear checked is because they don't have an established 'value.'  
i.e. `<Radio value={sampleValue} className=''>`
I was trying to brainstorm the best way to organize the value, and I'm not quite sure yet.  One way could be by doing
`value={{ hasRead: true, hasDrawn: false, hasWritten: false }}`, so on and so forth.

Also, I think we need one more radio option so users can have all tasks unchecked.

edit: here's the antd radio documentation; they give some examples that can be pretty useful to view, since you can view their code and see exactly how they used them: https://ant.design/components/radio/

Reply:
I found that documentation as well and was trying to figure out the best way to ad the value. I implemented the value to each of the buttons and added one more radio button, "A user has completed 0 tasks".

Reply:
It looks like to keep it controlled we'll need to add `checked` as well, but that will sort of depend on how we set up `value`

Reply:
Yep added that attribute